app/lib/common/actions/report_content.dart

```diff
@@ -101,10 +101,10 @@ class _ReportContentWidget extends ConsumerWidget {
                   style: Theme.of(context).textTheme.labelMedium,
                 ),
                 value: ref.watch(_ignoreUserProvider),
-                onChanged: (value) =>
-                    ref.read(_ignoreUserProvider.notifier).update(
-                          (state) => value!,
-                        ),
+                onChanged: (val) {
+                  if (val == null) throw 'Changed value not available';
+                  ref.read(_ignoreUserProvider.notifier).update((state) => val);
+                },
```

Comment:
why throw? throws in UI code are bad as they are often unrecoverable crashes if they do occur.
```suggestion
                  if (val == null) {
                     _log.severe('Changed value not available';
                     return
                  }
                  ref.read(_ignoreUserProvider.notifier).update((state) => val);
                },
```

Reply:
Understood.
But how to fix the case that return type is not `void` like [this fn](https://github.com/bitfriend/a3/blob/ce6ead4d8e39e186b294ac2511750b0fdfcf861c/app/lib/common/toolkit/buttons/danger_action_button.dart#L83-L88)?